src/templates/legal/terms_and_conditions.html.tt

```diff
@@ -37,7 +37,7 @@
         [% l('These Terms and Conditions, for the convenience of our Clients are provided in several different languages; however, these translations do reflect the same principles. We attempt, as much as possible to provide a faithful translation in English of the prevailing language. In the event of any differences between the English version of the website and any other language, the English version shall prevail.') %]
     </p>
     <p>
-        [% l("You may communicate with us through this website, by email or, exceptionally, by post. Details can be found in the Contact Us page.") %]
+        [% l("You may communicate with us through this website by means of Live Chat, by e-mail, by phone or, exceptionally, by post. Details can be found in the Contact Us page.") %]
```

Comment:
should say "You may communicate with us through this website by means of Live Chat, e-mail, phone or, exceptionally, by post."